scripts/seed-dummy-realtime.ts

```
@@ -0,0 +1,654 @@
+            'STARK',
+            ${Math.random() > 0.5},
+            true,
+            true
+          )
+          ON CONFLICT DO NOTHING
+          RETURNING *
+        `
+        cluster = result[0] || null
+      } else {
+        // Use Drizzle when columns exist
+        const clusterValues: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)